innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+* InnerSource Portal
+
+## Patlet
+
+InnerSource projects at your company are increasing but potential contributors have no easy way of discovering them.
+
+Create an intranet website that indexes all available InnerSource projects.  Make it easy for InnerSource projects owners to include information about their projects and for visitors to the site to search across all the available projects.  This will enable potential contributors to more easily locate projects of interests and learn more about who is running them, the project goals and access technical and community management related documentation.
+
+## Problem
+
+There is no central, easily accessible resource for InnerSource project owners and potential contributors to leverage to advertise or discover InnerSource projects.  
+
+## Story
+
+You are attempting to establish an InnerSource practice within your organization.  You are aware of some projects being run using an InnerSource model, but their existence is only being communicated via word of mouth, email or sidebar conversations with other employees.  There is no single, shared resource for employees across the organization to access which will allow them to easily discover all ongoing InnerSource projects.  This is severely limiting the growth potential of every InnerSource project.  What can be done to help all InnerSource projects raise their visibility to as large an audience as possible and attract contributors organization-wide?
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+
```

Comment:
Should there be a force around different repositories that might exist across the larger organization? Having multiple repositories could make searching for InnerSource projects more difficult.

Reply:
Yes.  There is a supposition here that a common source control system is in place and that it is permissible for groups within an organization to provide global read access to that source code.  This may not exist in some organizations but even if access is restricted to a specific sub-group it does not preclude the usefulness of a portal in that limited context.

Reply:
Putting on the developer hat for a moment: if I have no visibility into the project which is offering gigs and specifically the sources and the past collaboration, I am less likely to commit to a gig for that project.

Reply:
True, however remember that the premise is that developers are expected to be searching across the Gigs in the Marketplace to identify projects where they can either demonstrate existing or obtain new skills/experiences.  Not necessarily to find opportunities to contribute to projects of which they are already aware.